src/components/Item.js

```diff
@@ -24,16 +24,35 @@ const Item = ({ item, handleChange }) => {
     setChecked(check);
   }, [item]);
 
+  const addNextPurchaseStyling = (item) => {
+    let color = ''
+
+    if(item.status === 'inactive'){
```

Comment:
I love the color tone and styling of the list items. But I wasn't sure at first which color meant what. Would perhaps a red, yellow, green, and grey color scheme make sense?

Reply:
@AliLynne Thanks for your suggestion. Working on it.

Reply:
This is super smart! Was thinking that maybe we could have an object storing the color name to hex code mapping, and then set the color using something like `color = colorMap["red"]`